upgrade/helpers/tasks.py

```diff
@@ -887,3 +855,71 @@ def check_status_of_running_task(command, attempt):
         logger.info("Command: {}\n"
                     "Exceeded the maximum attempt to check the "
                     "tasks running status".format(command))
+
+
+def repository_setup(repository, repository_name, base_url, enable, gpgcheck):
+    """
+    This is generic fucntion which is used to setup the repository
+    :param repository: uniq repository ID
```

Comment:
Rename all docstrings as below:
```suggestion
    :param str repository: uniq repository ID
```
The parameter type in docstring is missing.

Reply:
good observation :+1: